runtimes/ruby-3.1/server.rb

```diff
@@ -3,117 +3,271 @@
 
 USER_CODE_PATH = '/usr/code-start';
 
-before do
-  content_type :json
-end
+class Response
```

Comment:
We should prefix the class names with `Runtime` for consistency across runtimes

Reply:
Updated.